examples/Dockerfile.centos8

```diff
@@ -0,0 +1,13 @@
+# ch-test-scope: standard
+FROM centos:8
+# Install the dnf ovl plugin to workaround rpmdb corruptions issues when using
+# overlayfs as the storage driver.
+# Enable the PowerTools repository as some packages in EPEL depend on it.
+RUN dnf -y install epel-release && dnf -y update \
+ && dnf -y install --enablerepo=epel-playground dnf-plugin-ovl \
+ && sed -i -e 's/enabled=0/enabled=1/g' /etc/yum.repos.d/CentOS-PowerTools.repo
```

Comment:
1. Format as:

```
RUN    dnf -y install epel-release \
    && dnf -y update \
    && dnf -y install --enablerepo=epel-playground dnf-plugin-ovl \
    && sed -i -e 's/enabled=0/enabled=1/g' /etc/yum.repos.d/CentOS-PowerTools.repo
```

(There are several places this general change should be made.)

2. Is it more idiomatic to say `dnf config-manager --set-enabled ...`?

Reply:
1. Will do

2. Turns out `config-manager` is part of the `dnf-plugins-core` package and not available by default. Since this is the only place we would use it I opted for the sed line instead. If you prefer installing that package and then using config manager more I will adjust.

Reply:
No, `sed` is good.